src/config/dynamic-theme-fixes.config

```diff
@@ -9682,6 +9682,21 @@ INVERT
 
 ================================
 
+pomofocus.io
+
+INVERT
+img[alt$="icon"]
+
+CSS
+div#target > div:first-child {
+    background-color: var(--darkreader-neutral-background) !important;
```

Comment:
Why should it be black? The original site uses a light red color, and thus dark reader correctly modifies to darker red color.

Reply:
i used the believe that the purpose of a fix were to implement a dark background for all colors, if that isn't the case then you can close this pr

Reply:
The dynamic fix config is to fix websites that have aspects that dark reader can't handle or incorrectly handle.